bigquery/cloud-client/authorized_view_tutorial_test.py

```diff
@@ -24,24 +25,24 @@ def client():
 
 
 @pytest.fixture
-def to_delete(client):
+def datasets_to_delete(client):
     doomed = []
     yield doomed
     for item in doomed:
-        if isinstance(item, (bigquery.Dataset, bigquery.DatasetReference)):
-            client.delete_dataset(item, delete_contents=True)
-        elif isinstance(item, (bigquery.Table, bigquery.TableReference)):
-            client.delete_table(item)
-        else:
-            item.delete()
+        client.delete_dataset(item, delete_contents=True)
```

Comment:
Is this client automatically retrying transient errors? If not, what happens if delete fails?

Reply:
Yes, commented elsewhere this library has a default retry policy.